docs/DeveloperGuide.md

```diff
@@ -509,23 +524,35 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 2. HelloFile shows the user help.
 
     Use case ends.
-
+    
 **Extensions**
 
-* *a. User input the wrong command.
-	
-	* *a1. HelloFile shows the error message and direct the user to enter the user help command.
-	
-	    Use case ends.
 
+	* 2a1. HelloFile shows the error message.
+		
+	    Use case resumes from step 1.
+	        
+* *a. HelloFile has no permission to read or write to the file system.
```

Comment:
I think HelloFile does not need to read or write in help command, it shouldn't be in extension.

Reply:
This is the same as Luo yi's comment at the top.

Reply:
should I remove the whole extension?